index.html

```diff
@@ -8,8 +8,39 @@
 <div class="block">
   <h1><span class="header-inside">Projects</span></h1>
   <div id="projects-panel">
+    <table class="projects">
+      <noscript>
+        {% for p in site.data.projects %}
+        {% assign project = p[1] %}
+        <tbody>
+          <tr>
+            <td colspan="2" class="title">
+              <span class="proj"><a href="{{project.site}}">{{project.name}}</a></span>
+            </td>
+          </tr>
+          <tr>
+            <td class="details">
+              <p class="label"><a href="{{ project.upforgrabs.link }}" title="View open issues for {{ project.name }}">{{ project.upforgrabs.name }}</a></p>
+            </td>
+            <td class="details">
+              {% if project.desc %}
+                <span class="desc">{{ project.desc }}</span>
+              {% endif %}
+              {% if project.tags %}
+                <p class="tags">
+                  {% for tag in project.tags %}
+                    {{ tag }}{% if forloop.index != forloop.length %},{% endif %}
```

Comment:
There should be a space after that comma, else you're going to see a list like this:  
tag1,tag2,tag3

For future reference, you can simplify that if statement like this:

```html
{{ tag }}{% if not forloop.last %}, {% endif %}
```